lib/auth/webauthnwin/ctypes.go

```diff
@@ -61,9 +59,11 @@ type webauthnAuthenticatorMakeCredentialOptions struct {
 	dwVersion             uint32
 	dwTimeoutMilliseconds uint32
 	// Credentials used for exclusion.
-	CredentialList webauthnCredentials
+	//nolint:unused
```

Comment:
i needed to add that //nolint:unused annotations. This types reflect API types however few fields are not used. Also linter does not run inside windows build tags, that's why some incorrectly reported as unused.